stdlib/3/unittest.pyi

```diff
@@ -15,8 +15,8 @@ _FT = TypeVar('_FT', Callable[[Any], Any])
 
 
 def skip(reason: str) -> Callable[[_FT], _FT]: ...
-def skipIf(condition: bool, reason: str) -> Callable[[_FT], _FT]: ...
-def skipUnless(condition: bool, reason: str) -> Callable[[_FT], _FT]: ...
+def skipIf(condition: object, reason: str) -> Callable[[_FT], _FT]: ...
+def skipUnless(condition: object, reason: str) -> Callable[[_FT], _FT]: ...
```

Comment:
I think it's better to always write down a type in a stub when you know what it is, because otherwise it's not clear if the absence of a type is purposeful or if the stub is incomplete.  If we left the type out here, for example, I could imagine someone in the future trying to change it back to `bool`, thinking it was accidentally left out.

Also, I'd strongly prefer `object` to `Any` in this case, even though they have the same effect: `Any` implies the use of an escape hatch is necessary/the function can't be properly typed, which isn't the case here.